modules/eks/argocd/main.tf

```diff
@@ -43,6 +43,13 @@ locals {
         value = nonsensitive(local.webhook_github_secret)
         type  = "string"
       }
+    ] : [],
+    local.slack_notifications_enabled ? [
+      {
+        name  = "notifications.secret.items.slack-token"
+        value = nonsensitive(data.aws_ssm_parameter.slack_notifications[0].value)
```

Comment:
why is this being changed to nonsensitive here? if this is sensitive, it should still be able to be used as a kv in the argocd template without needing to be `nonsensitive`

Reply:
good call. removed and tested without